src/resources/postcss/views/full/_single-event.pcss

```diff
@@ -81,4 +81,37 @@
 	@media (--viewport-medium) {
 		@mixin desktop-heading-6;
 	}
-}
\ No newline at end of file
+}
+
+/* Description Block */
+.tribe-events-content {
+	@mixin body;
+	
+	h2, h3, h4, h5, h6 {
+		@mixin heading;
+		letter-spacing: normal;
+		text-transform: none;
+	}
+	
+	h2 {
+		@mixin mobile-heading-3;
+	}
```

Comment:
should we have more specific styles for `h3` - `h6`?

Reply:
Rachel said they're good for now, will maybe tackle any further change during DQA